app/controllers/branches_controller.rb

```diff
@@ -0,0 +1,45 @@
+# Controller to handle model Branch CRUD
+class BranchesController < ApplicationController
+  before_action :set_branch, only: %i[ show edit update destroy ]
+
+  def index
+    @branches = Branch.all
+  end
+
+  def new
+    @branch = Branch.new
+  end
+
+  def create
+    @branch = Branch.new(branch_params)
+
+    if @branch.save
+      redirect_to branches_url, notice: t('.success')
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def update
+    if @branch.update(branch_params)
+      redirect_to branches_url, notice: t('success')
```

Comment:
`.` missing that's why translation are not working.
should be `.success`

Reply:
Done